docs/website/content/docs/v0.3/Configuration/servers.md

```diff
@@ -77,7 +77,7 @@ Sidero will never write to or wipe any disk on a computer which is not marked as
 This can be tedious for systems in which all attached computers should be considered to be under the control of Sidero.
 Thus, you may also choose to automatically accept any machine into Sidero on its discovery.
 Please keep in mind that this means that any newly-connected computer **WILL BE WIPED** automatically.
-You can enable auto-acceptance by pasing the `--auto-accept-servers=true` flag to `sidero-controller-manager`.
+You can enable auto-acceptance by passing the `--auto-accept-servers=true` flag to `sidero-controller-manager`.
```

Comment:
we should also document the new variable in the `Geting Started/installation.md` document

Reply:
alrighty @smira, think I've got your feedback taken care of.